testdata/baselines/reference/submodule/compiler/expressionWithJSDocTypeArguments.errors.txt

```diff
@@ -43,79 +17,27 @@ expressionWithJSDocTypeArguments.ts(27,39): error TS1109: Expression expected.
                          ~
 !!! error TS1110: Type expected.
     const HuhFoo = foo<string?>;

```

Comment:
I thought this was an intentional difference? https://github.com/microsoft/typescript-go/blob/main/CHANGES.md#parser:~:text=No%20postfix%20T%3F%20and%20T!%20types. notes that as a removed thing.

Reply:
Hadn't noticed that we were going to outlaw the postfix forms. Let's discuss and if we still feel that way we can back it out. Or we could continue to parse them (for better error recovery) and then always issue a grammar error.